src/platform/linux.rs

```diff
@@ -1073,13 +1073,11 @@ mod desktop {
         }
 
         pub fn refresh(&mut self) {
-            let seat0_values = get_values_of_seat0(&[0, 1, 2]);
-            if !self.sid.is_empty() && is_active(&self.sid) {
-                if self.sid == seat0_values[0] {
+            if !self.sid.is_empty() && is_active_and_seat0(&self.sid) {
                      return;
-                 }
             }
 
+            let seat0_values = get_values_of_seat0(&[0, 1, 2]);
```

Comment:
Back to it's original place just like how it was before https://github.com/rustdesk/rustdesk/commit/ebfebdffa7a23c66f7aa508f9fb02e0dedb7acc5.